tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -53,6 +53,8 @@ class ConnectDistributedTest(Test):
     STATUS_TOPIC = "connect-status"
     STATUS_REPLICATION_FACTOR = "1"
     STATUS_PARTITIONS = "1"
+    SCHEDULED_REBALANCE_MAX_DELAY_MS = "60000"
```

Comment:
Does this need to be so large by default?

Reply:
So, we are trying to strike a balance here. We don't want this to be too high, because tests will start failing since tasks will remain unassigned, but we also want to exercise the delay logic because workers do indeed come back. Usually with class loading and stuff it takes less than 30 seconds for a worker to come back and join the group. But to guard against flakiness I'm making this 60sec. 

In two places the delay will need to increase but only for tests setting protocol to `compatible`. In one the wait time increased from 70sec to 150sec and another case from 30sec to 120sec. Also, this applies to worst case scenarios when a real failure happens. In normal times, the assertions will take a bit less than that, but indeed more than 60sec. 

Would you prefer another value?

Reply:
Your description was very helpful. No need to change the value.